package.json

```diff
@@ -7,24 +7,23 @@
     "webtreemap": "build/src/cli.js"
   },
   "dependencies": {
-    "commander": "^2.11.0",
+    "commander": "^6.1.0",
     "open": "^7.2.1",
     "tmp": "^0.2.1"
   },
   "devDependencies": {
-    "@types/commander": "^2.11.0",
-    "@types/node": "^8.0.34",
-    "@types/open": "^6.2.1",
+    "@types/commander": "^2.12.2",
+    "@types/node": "^14.6.4",
     "@types/tmp": "^0.2.0",
     "prettier": "1.14.3",
-    "typescript": "^3.1.3",
+    "typescript": "^4.0.2",
     "webpack": "^3.7.1"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "fmt": "prettier --write src/*.ts demo/*.ts *.md",
     "build": "tsc && webpack -p",
-    "demo": "du -a node_modules/ | awk '{print $1*512 \"\t\" $2}' | node build/src/cli.js --title 'node_modules for webtreemap' > docs/index.html"
+    "demo": "du -a node_modules/ | awk '{print $1*512 \"\t\" $2}' | node build/src/cli.js --title 'node_modules for webtreemap'"
```

Comment:
Prolly should be in previous commit, but I don't care too much.

Reply:
You should use [Squash & Merge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-merges#squash-and-merge-your-pull-request-commits) and not worry about PR branch commits :)